consensus/consensus-types/src/sync_info.rs

```diff
@@ -47,7 +44,7 @@ impl SyncInfo {
     pub fn new_decoupled(
         highest_quorum_cert: QuorumCert,
         highest_ordered_cert: QuorumCert,
-        highest_ledger_info: Option<LedgerInfoWithSignatures>,
+        highest_commit_cert: Option<QuorumCert>,
```

Comment:
let's make it non-option, i think the option is there to provide backward compatibility

Reply:
we seem to be setting it to None here - https://github.com/aptos-labs/aptos-core/blob/ac9eeaf35d7d18a09afda38f92e1e563f19fa333/consensus/consensus-types/src/sync_info.rs#L55, so I can remove it from this function, but not from SyncInfo struct?

Reply:
this Option is for highest ordered cert but not commit_cert?